src/peep/peep.c

```diff
@@ -7499,7 +7505,7 @@ rct_map_element* get_banner_on_path(rct_map_element *path_element)
 
 static int banner_clear_path_edges(rct_map_element *mapElement, int edges)
 {
-	if (RCT2_GLOBAL(0x00F1AEDD, uint8) & 0x80) return edges;
+	if (_peepPathFindIsStaff == true) return edges;
```

Comment:
I thought you preferred it to be explicit but i see now you were mainly talking about NULL checks being explicit.